packages/design-system/src/components/OcSelect/OcSelect.vue

```diff
@@ -8,12 +8,12 @@
       class="oc-select"
       style="background: transparent"
       v-bind="additionalAttributes"
-      v-on="$listeners"
+      @update:modelValue="$emit('update:modelValue', $event)"
```

Comment:
Maybe could be achieved with a compat config as well (so lzsteners are included ln attrs)

Reply:
Couldn't make that work